src/Infrastructure/Repositories/AppUserRepository.cs

```diff
@@ -11,9 +12,30 @@ namespace Infrastructure.Repositories
 {
     public class AppUserRepository : GenericRepository<AppUser>, IAppUserRepository
     {
-        public AppUserRepository(ApplicationDbContext context) : base(context)
+        public AppUserRepository(ApplicationDbContext context) : base(context) { }
+
+        public override async Task<List<AppUser>> GetAllAsync()
         {
+            return await _context.AppUsers.AsNoTracking()
+                .OrderBy(b => b.LastName)
+                .ToListAsync();
+        }
 
+        public override async Task<AppUser> GetByIdAsync(Guid id)
+        {
+            return await _context.AppUsers.AsNoTracking()
```

Comment:
here u can remove where and firstordefault then put .FindAsync(id) method

Reply:
We deleted this overridden method

Reply:
Changes done in Generic Repository